docker/centos/Dockerfile

```diff
@@ -35,7 +35,8 @@ ENV MINIFI_BASE_DIR /opt/minifi
 RUN mkdir -p $MINIFI_BASE_DIR 
 USER $USER
 
-RUN yum -y install java-1.8.0-openjdk java-1.8.0-openjdk-devel gcc g++ sudo git which maven
+RUN yum -y install java-1.8.0-openjdk java-1.8.0-openjdk-devel gcc g++ sudo git which maven make
+RUN yum -y install openssl openssl-devel openssl-static
```

Comment:
This is intended to be temporary until the third party compilations are fixed in MINIFICPP-1022

Reply:
Let's wait on 1022, then

Reply:
Agreed, removed the hotfix.